index.js

```diff
@@ -196,14 +209,21 @@ function colorDelta(img1, img2, k, m, yOnly) {
         b2 = blend(b2, a2);
     }
 
-    const y = rgb2y(r1, g1, b1) - rgb2y(r2, g2, b2);
+    const y1 = rgb2y(r1, g1, b1);
+    const y2 = rgb2y(r2, g2, b2);
+    const y = y1 - y2;
+
+    if (y1 > y2) origin = 2;
```

Comment:
I'm not sure how bulletproof just comparing the brightness factor is but it passes my test case.

Reply:
Yeah, looks like it will only work as intended for dark-on-light images with a clear background, and may get weird for the general case. Perhaps it's worth rephrasing the description of the coloring options then.